spec/features/integrated_application/integrated_application_with_single_member_spec.rb

```diff
@@ -1,7 +1,7 @@
 require "rails_helper"
 
 RSpec.feature "Integrated application" do
-  scenario "with one member", :a11y do
+  scenario "with one member", :js do
```

Comment:
Remove `:a11y` means this feature spec doesn't check for accessibility. I don't think that's ideal, but it's more ideal than making a duplicate spec that does.

Reply:
yeah not ideal, but I think better than the alternative